skippy-gradle-android/src/main/java/io/skippy/gradle/android/ProjectSettings.java

```diff
@@ -27,17 +33,31 @@ private ProjectSettings(List<File> classesDirs, SkippyPluginExtension skippyExte
     }
 
     static ProjectSettings from(Project project) {
+        BaseExtension androidExtension = project.getExtensions().findByType(BaseExtension.class);
+        Stream<AndroidSourceSet> androidSourceSets = androidExtension.getSourceSets()
+                .stream()
+                .map(deprecatedAndroidSourceSet -> deprecatedAndroidSourceSet);
+
+        List<File> classesDirs = AndroidTestSourceSetCollector.collectIfExists(androidSourceSets).toList();
```

Comment:
`classesDirs` needs to point to the directories that contain the compiled class files.

Note to myself: I have to document that better.

Reply:
@Link184 @fmck3516 
For an Android project using Kotlin, that would be build/tmp/kotlin-classes/<sourceset> if I rwcall that correctly